src/routing.js

```diff
@@ -34,8 +34,8 @@ const _applySerialization = (stateData, map) => {
     return result;
 };
 
-export const stateToQueryString = (state, map) => {
-    const serializedState = _pruneNullLeaves(_applySerialization(state, map));
+export const stateToQueryString = (state, maps) => {
```

Comment:
What about using `...maps` or allowing both a list or a single argument? Like this simple cases don't need to start providing a list but you still have the option to do so if needed